assets/js/components/surveys/SurveyQuestionRatingChoice.js

```diff
@@ -59,8 +67,15 @@ const SurveyQuestionRatingChoice = ( { Icon, choice, answerQuestion } ) => {
 	);
 };
 
+SurveyQuestionRatingChoice.ordinalIconMap = {
```

Comment:
Why not just a `const` ? 🤔 

```suggestion
const ordinalIconMap = {
```

Reply:
👍